packages/api/src/services/popular_reads.ts

```diff
@@ -88,6 +95,51 @@ export const addPopularRead = async (
   return pageId
 }
 
+const addPopularReads = async (
+  userId: string,
+  names: string[]
+): Promise<AddPopularReadResult[]> => {
+  const results: AddPopularReadResult[] = []
+  for (const name of names) {
+    const pageId = await addPopularRead(userId, name)
+    results.push({
+      pageId,
+      name,
+      status: pageId
+        ? ArticleSavingRequestStatus.Succeeded
+        : ArticleSavingRequestStatus.Failed,
+    })
+  }
+  return results
+}
+
+export const addPopularReadsForNewUser = async (
+  userId: string
+): Promise<void> => {
+  const defaultReads = [
+    'omnivore_get_started',
+    'power_read_it_later',
+    'omnivore_organize',
+  ]
+
+  // get client from request context
+  const client = httpContext.get('client') as string | undefined
+
+  switch (client) {
```

Comment:
For now, because we don't have these guides, these will just fail to add, but nothing bad will happen I think.

Reply:
Yes, we will simply ignore these nonexistent guide